portfolio/src/main/webapp/contact.html

```diff
@@ -26,9 +26,9 @@ <h2>
                 data-rotate='["Contact" ]'></span>
             </h2>
             <div class="social-icon">
-                <a class="icon" href="https://www.linkedin.com/in/cocosack/"> <i class="icon-linkedin-sign icon-3x"></i></a>
-                <a class="icon" href="https://github.com/cocovsack/step_portfolio"> <i class="icon-github-sign icon-3x"></i></a>
-                <a class="icon" href="mailto: cocosack@gmail.com"> <i class="icon-envelope icon-3x"></i></a>
+                <a id="icon" href="https://www.linkedin.com/in/cocosack/"> <i id="icon" class="icon-linkedin-sign icon-3x"></i></a>
+                <a id="icon" href="https://github.com/cocovsack/step_portfolio"> <i id="icon" class="icon-github-sign icon-3x"></i></a>
+                <a id="icon" href="mailto: cocosack@gmail.com"> <i id="icon" class="icon-envelope icon-3x"></i></a>
```

Comment:
Id should be unique per html element. Could you use class instead?

Also, indent by 2 spaces a time.